test/unit/hostgroup_test.rb

```diff
@@ -345,7 +345,7 @@ def setup_user(operation)
 
   test "hostgroup name can't be too big to create lookup value matcher over 255 characters" do
     parent = FactoryGirl.create(:hostgroup)
-    min_lookupvalue_length = "hostgroup=".length + parent.title.length + 1
+    min_lookupvalue_length = 10 + parent.title.length + 1
```

Comment:
I think it's quite important to keep this test code readable, else somebody is going to change it without realising why it's testing the validator in this way.  At the very least it must be commented, but I'd suggest excluding the cop on these methods.